README.md

```diff
@@ -1 +1 @@
-# protoc-gen-preprocess
\ No newline at end of file
+# protoc-gen-preprocess
```

Comment:
Please add some details in readme about what this plugins does and how to use it.

Reply:
Added readme and example application